SU-HKKU/cardinal_service_profile
# CSD-71 Migration configuration and fields for courses from explorecourses

# READY FOR REVIEW

# Summary
- Migration entities for course importers into opportunities.

# Need Review By (Date)
- 5/19

# Urgency
- Medium

# Steps to Test
1. `composer require su-hkku/cardinal_service_profile:dev-CSD-71`
1. `blt drupal:sync --site=cardinalservice -n`
1. run one of the migrations like `drush @cardinalservice.local mim cardcourses_educ`
1. in the admin, verify some courses were imported
1. view the course to verify alot of fields were populated.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
